cyral/manifest.json

```
@@ -0,0 +1,32 @@
+{
+  "display_name": "Cyral",
+  "maintainer": "product@cyral.com",
+  "manifest_version": "1.0.0",
+  "name": "cyral",
+  "metric_prefix": "cyral.",
+  "metric_to_check": "",
```

Let's include a metric to check, this will automatically install the Cyral tile when detected on the Integrations page.

```suggestion
  "metric_to_check": "cyral.analysis_time",
```